### DISCUSSION TITLE
Ability to use Statamic forms

### DISCUSSION THREAD

**Discussion by robdekort** [duncanmcclean/guest-entries#57] [Feature Requests]
It would be really cool if you could use Statamic forms to save guest entries. 

Benefits:

- Very easy to create the frontend forms.
- Folks often already have the rendering of forms dynamically in place so it saves quite some dev time.
- Form code would by dry'er as you don't have to double style form fields. 
- You can use conditional logic.
- Folks could even hook it up to something like Livewire Forms.

**Comment by wm-simon** [comment]
Are there any plans to get this?

**Reply by duncanmcclean** [reply]
No immediate plans, no.

**Comment by robdekort** [comment]
In addition to this it would probably be nice if those forms would support Precognition as well.

This is just me dreaming though.
